### DISCUSSION TITLE
I'm having an issue getting my application approved

### DISCUSSION THREAD

**Discussion by dohertysam77** [etsy/open-api#860] [Q&A]
I always get invalid website URL. I'm not sure what they are looking for here. I don't have a developer website and my shop is not integrated with another website (wordpress, shopify, ect). What should I enter for this field?

![Capture](https://user-images.githubusercontent.com/124935842/217944867-0a8f068b-8d22-4f05-abcb-828d9615a562.JPG)

**Comment by Gareth064** [comment]
So is your app like a background service with no UI?
If so then you need to explain the very well in the Description text. Then maybe just put https://localhost.com or something like that in the Website URL.

**Reply by dohertysam77** [reply]
Exactly, I just want to collect data from search results and export to a csv.
I'll give that a try.
Thank you!
